BrowserKit/Sources/Common/Logger/LoggerFileManager.swift

```diff
@@ -37,15 +40,25 @@ class DefaultLoggerFileManager: LoggerFileManager {
     }
 
     func copyLogsToDocuments() {
-        deleteOldLogs()
+        deleteOldDocumentDirectoryLogs()
         copyLogs()
     }
 
     // MARK: - Private
 
+    /// Deletes all log files in Caches (and Documents) directory.
+    func deleteCachedLogFiles() {
```

Comment:
Can we add a test case in `LoggerFileManagerTests` for this `deleteCachedLogFiles`?

Reply:
Yes good call-out thanks, I added an initial test for now and am taking a closer look at possibly building those out a bit further as well